boot/zephyr/main.c

```diff
@@ -67,7 +67,7 @@ const struct boot_uart_funcs boot_funcs = {
 #define ZEPHYR_LOG_MODE_MINIMAL 1
 #endif
 
-#if defined(CONFIG_LOG) && !defined(CONFIG_LOG_IMMEDIATE) && \
+#if defined(CONFIG_LOG) && !defined(CONFIG_LOG_MODE_IMMEDIATE) && \
```

Comment:
can https://github.com/mcu-tools/mcuboot/pull/1327 content be used instead of fix here?